bin/tictactoe

```diff
@@ -1,3 +1,24 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
```

Comment:
Thanks for the feedback Cernan, I probably should have made the time to take my learnings from the gem project back to the ttt project. I have implemented the cli class and moved the game logic to it.